### YamlMime:ManagedReference
items:
- uid: System.Data.Services.Client.DataServiceQueryException
  id: DataServiceQueryException
  children:
  - System.Data.Services.Client.DataServiceQueryException.#ctor
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.String)
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception)
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)
  - System.Data.Services.Client.DataServiceQueryException.Response
  langs:
  - csharp
  name: DataServiceQueryException
  nameWithType: DataServiceQueryException
  fullName: System.Data.Services.Client.DataServiceQueryException
  type: Class
  summary: "从数据服务加载属性值时出错，该值指示错误的异常。"
  remarks: "调用方应检查<xref:System.Data.Services.Client.DataServiceQueryException.Response%2A>以获得异常结果。</xref:System.Data.Services.Client.DataServiceQueryException.Response%2A>"
  syntax:
    content: >-
      [System.Diagnostics.DebuggerDisplay("{Message}")]

      public sealed class DataServiceQueryException : InvalidOperationException
  inheritance:
  - System.InvalidOperationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor
  id: '#ctor'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException()
  nameWithType: DataServiceQueryException.DataServiceQueryException()
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException()
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "初始化的新实例<xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;></xref>使用系统提供的描述错误的消息的类。"
  remarks: "此构造函数初始化<xref:System.Exception.Message%2A>属性为系统提供的消息的描述错误的新实例。</xref:System.Exception.Message%2A> 此消息将当前系统区域性考虑在内。       下表显示<xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>实例的初始属性的值      |属性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|本地化的错误消息字符串。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceQueryException ();
    parameters: []
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(String)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "初始化的新实例<xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;></xref>使用指定的描述错误的消息的类。"
  remarks: "下表显示<xref:System.Data.Services.Client.DataServiceQueryException>.</xref:System.Data.Services.Client.DataServiceQueryException>实例的初始属性的值      |属性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|中指定的错误消息字符串`message`。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceQueryException (string message);
    parameters:
    - id: message
      type: System.String
      description: "描述异常的消息。 此构造函数的调用方需要确保此字符串已针对当前系统区域性进行了本地化。"
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(SerializationInfo,StreamingContext)
  nameWithType: DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  syntax:
    content: protected DataServiceQueryException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "要添加。"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "要添加。"
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(String,Exception)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "初始化的新实例<xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;></xref>使用指定的错误消息和对导致此异常的内部异常的引用的类。"
  remarks: "前一个异常的直接结果应包含中的前一个异常的引用引发的异常，<xref:System.Exception.InnerException%2A>属性。</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>属性返回相同的值传递到构造函数中，或`null`如果<xref:System.Exception.InnerException%2A>属性不会提供给构造函数的内部异常值。</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       下表显示<xref:System.Data.Services.Client.DataServiceQueryException>.</xref:System.Data.Services.Client.DataServiceQueryException>实例的初始属性的值      |属性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|中指定的错误消息字符串`message`。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceQueryException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "描述异常的消息。 此构造函数的调用方需要确保此字符串已针对当前系统区域性进行了本地化。"
    - id: innerException
      type: System.Exception
      description: "导致当前异常的异常。 如果`innerException`参数不是<xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>中, 引发当前异常<xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;></xref>处理内部异常的块。"
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)
  id: '#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(String,Exception,QueryOperationResponse)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "初始化的新实例<xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;></xref>类。"
  syntax:
    content: public DataServiceQueryException (string message, Exception innerException, System.Data.Services.Client.QueryOperationResponse response);
    parameters:
    - id: message
      type: System.String
      description: "包含错误消息的字符串值。"
    - id: innerException
      type: System.Exception
      description: "内部异常对象中。"
    - id: response
      type: System.Data.Services.Client.QueryOperationResponse
      description: "<xref href=&quot;System.Data.Services.Client.QueryOperationResponse&quot;> </xref>对象。"
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.Response
  id: Response
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: Response
  nameWithType: DataServiceQueryException.Response
  fullName: System.Data.Services.Client.DataServiceQueryException.Response
  type: Property
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "获取<xref href=&quot;System.Data.Services.Client.QueryOperationResponse&quot;> </xref> ，该值指示异常结果。"
  syntax:
    content: public System.Data.Services.Client.QueryOperationResponse Response { get; }
    return:
      type: System.Data.Services.Client.QueryOperationResponse
      description: "A <xref href=&quot;System.Data.Services.Client.QueryOperationResponse&quot;> </xref>指示异常结果的对象。"
  overload: System.Data.Services.Client.DataServiceQueryException.Response*
  exceptions: []
  platform:
  - net462
references:
- uid: System.InvalidOperationException
  isExternal: false
  name: System.InvalidOperationException
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException()
  nameWithType: DataServiceQueryException.DataServiceQueryException()
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException()
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(String)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(SerializationInfo,StreamingContext)
  nameWithType: DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(String,Exception)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(String,Exception,QueryOperationResponse)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
- uid: System.Data.Services.Client.QueryOperationResponse
  parent: System.Data.Services.Client
  isExternal: false
  name: QueryOperationResponse
  nameWithType: QueryOperationResponse
  fullName: System.Data.Services.Client.QueryOperationResponse
- uid: System.Data.Services.Client.DataServiceQueryException.Response
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: Response
  nameWithType: DataServiceQueryException.Response
  fullName: System.Data.Services.Client.DataServiceQueryException.Response
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor*
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException
  nameWithType: DataServiceQueryException.DataServiceQueryException
- uid: System.Data.Services.Client.DataServiceQueryException.Response*
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: Response
  nameWithType: DataServiceQueryException.Response
